src/DependencyInjection/HttplugExtension.php

```diff
@@ -60,11 +59,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
         $loader->load('services.xml');
         // TODO: Move this back into services.xml when we drop support for Symfony 4, or completely remove the service in the next major version.
-        if (Kernel::MAJOR_VERSION >= 5) {
-            $loader->load('services_legacy.xml');
-        } else {
-            $loader->load('services_legacy_sf4.xml');
-        }
+        $loader->load('services_legacy.xml');
```

Comment:
we could completely remove the legacy services and mention in the changelog what the new names to use are. or prefer to do that in a separate MR?

Reply:
Sure, I can do this here. I take it, this means to also remove [the service alias in the services.xml](https://github.com/php-http/HttplugBundle/blob/1b16d97ad8d572211f61135139b367bf47e7d7ec/src/Resources/config/services.xml#L50)?

Reply:
`Psr\Http\Client\ClientInterface` should still be autowireable. so i think that should stay.

Reply:
I think I misunderstood what this when I first adjusted the xml config. Both legacy XML files triggered a deprecation if `Http\Client\HttpClient` was used. I first read this the wrong way and thought it created a `httplug.client` alias.